docs/dgeni.conf.js

```diff
@@ -31,12 +33,26 @@ package.config(function(log, readFilesProcessor, templateFinder, writeFilesProce
     basePath: 'src'
   }];
 
+
+  // Nunjucks and Angular conflict in their template bindings so change the Nunjucks
+  templateEngine.config.tags = {
+    variableStart: '{$',
+    variableEnd: '$}'
+  };
+
+  templateEngine.filters.push(require('./rendering/relativeLinkInlineTag'));
```

Comment:
Tell dgeni about our little helper tag that generates a link that is relative to the current doc.
In the main AngularJS app we don't need this since we are running an Angular app using routes that has its base href set.